syntax = "proto3";
package plugin;





/*
UpdatePlugin请求
*/
message UpdatePluginRequest {
    /*
    ID
    */
    string id = 1 ;
    /*
    名称
    */
    string name = 2 ;
    /*
    备注
    */
    string memo = 3 ;
    /*
    部署路径
    */
    repeated string deployPath = 4 ;
}

/*
UpdatePlugin返回
*/
message UpdatePluginResponse {
    /*
    插件ID
    */
    string id = 1 ;
}

/*
UpdatePluginApi返回
*/
message UpdatePluginResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    plugin.UpdatePluginResponse data = 4 ;
}

